pywt/_extensions/c/wavelets.c

```diff
@@ -291,7 +314,14 @@ Wavelet* wavelet(char name, unsigned int order)
             w->compact_support = 1;
             w->family_name = "Discrete Meyer (FIR Approximation)";
             w->short_name = "dmey";
-
+            w->dwt_possible = 1;
+            w->cwt_possible = 1;
```

Comment:
Again, this would be nicer if it was split over two different structs. That way it'd be possible to catch some errors at compile time if the wrong type is used.